scripts/dy_services_helpers/requirements.txt

```diff
@@ -0,0 +1,2 @@
+tenacity~=4.12
```

Comment:
Please make sure it is in sync throughout the entire repo ... 
By default it should be like that to reduce maintenance time

Reply:
done